app/forms/instance-create.tsx

```diff
@@ -95,11 +106,23 @@ const baseDefaultValues: InstanceCreateInput = {
   userData: null,
 }
 
+const useBootDiskItems = (projectSelector: PP.Project) => {
+  const { data: disks } = useApiQuery('diskList', {
+    query: { ...projectSelector, limit: 1000 },
+  })
+  return (
+    disks?.items
+      .filter(diskCan.attach)
```

Comment:
We might want to include disks that cannot be attached, or add a tooltip that lets the user know which disks are included and why.